From 78bdd2cce18f4f9b60ba8d16ea9eef74e66df515 Mon Sep 17 00:00:00 2001
From: Rob Clark <rob@ti.com>
Date: Thu, 19 Aug 2010 10:32:52 -0500
Subject: [PATCH 20/26] playbin: disable interlaced support

Latest totem is enabling interlaced support, which causes similar issues
to when native-video is not used.. for now, since none of the codecs
support it, disable interlaced support.
---
 gst/playback/gstplaysink.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/gst/playback/gstplaysink.c b/gst/playback/gstplaysink.c
index 969a785..91541ed 100644
--- a/gst/playback/gstplaysink.c
+++ b/gst/playback/gstplaysink.c
@@ -2027,11 +2027,13 @@ gst_play_sink_reconfigure (GstPlaySink * playsink)
     /* we have video and we are requested to show it */
     need_video = TRUE;
 
+#if 0
     /* we only deinterlace if native video is not requested and
      * we have raw video */
     if ((flags & GST_PLAY_FLAG_DEINTERLACE)
         && !(flags & GST_PLAY_FLAG_NATIVE_VIDEO) && playsink->video_pad_raw)
       need_deinterlace = TRUE;
+#endif
   }
 
   if (playsink->audio_pad) {
-- 
1.7.1

